Client/Client.csproj

```diff
@@ -79,4 +80,10 @@ This release is based on the Zeebe 0.24.X release (https://github.com/zeebe-io/z
     </AssemblyAttribute>
   </ItemGroup>
 
+  <ItemGroup>
+    <Reference Include="System.Threading.Tasks.Dataflow">
```

Comment:
I think this can be removed right?

Reply:
Not sure, had problems in my build. But if your build system works fine, sure.